onnxruntime/core/providers/webnn/builders/helper.h

```diff
@@ -203,11 +203,11 @@ std::unordered_set<const Node*> GetSupportedNodes(const GraphViewer& graph_viewe
                                                   const emscripten::val& wnn_limits,
                                                   const logging::Logger& logger);
 
-// Retrieve the first input name of a WebNN op used for validating supported input data types.
+// Retrieve the first input name of an ONNX op's corresponding WebNN op used for validating supported input data types.
 // WebNN ops have various first input names such as 'a', 'input', 'inputs', etc.
 // All WebNN op inputs are recorded in op_inputs_map.
-inline std::string_view GetWebNNOpFirstInputName(const std::string_view webnn_op_type) {
-  auto it = op_inputs_map.find(webnn_op_type);
+inline std::string_view GetWebNNOpFirstInputName(const std::string_view op_type) {
```

Comment:
Suggest parameter name `onnx_op_type` to make it clearer, since the function is still named "Get**WebNN**Op" (and so people might expect you get the WebNN op's first input name).

Reply:
Good suggestion, done.